pandas/core/indexes/base.py

```diff
@@ -2588,6 +2598,10 @@ def get_indexer(self, target, method=None, limit=None, tolerance=None):
         target = _ensure_index(target)
         if tolerance is not None:
             tolerance = self._convert_tolerance(tolerance)
+            if isinstance(tolerance, np.ndarray) and \
+               target.size != tolerance.size and tolerance.size > 1:
```

Comment:
use len

Reply:
went back to size since you cannot call len() on scalar arrays.